src/components/Conversion/NatureOfBusiness.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+  <section id="nature-of-business-section">
+  </section>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
+export default class NatureOfBusiness extends Vue {
+  // FUTURE
```

Comment:
:thumbsup: NatureOfBusiness and BusinessStartDate components do not allow modifications in the current edit UI ( with url /change), but they can bemodified in create ui. That's why they are included for the conversion filing.

Reply:
Yes, good catch.

I really don't know what those will look like in the end. Maybe we won't even need these components. We have a ticket for each of them, so this is future work.